__mocks__/styleMock.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
what is styleMock?

Reply:
in order to avoid importing `scss` files, we load this instead. which is just an empty module.